snowpack/src/build/build-import-proxy.ts

```diff
@@ -213,7 +237,9 @@ export async function wrapImportProxy({
   config: SnowpackConfig;
 }) {
   const {baseExt, expandedExt} = getExt(url);
-
+  if (baseExt === '.wasm') {
```

Comment:
To handle the wasm from `node_modules` usage, do not check `typeof code === 'string'` , 
it'll be `object` in this use case, not quite sure why it's not happened in `.css` case.

Reply:
In some cases we do return `object` when we’re handing sourcemaps. Not sure if that applies here or not, but something we could probably do a better job of clarifying for sure.